examples/multiple-metric-readers.js

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
I don't think this example is necessary. It's a list of metric readers, that should be fairly self-explainatory. :)